corgidrp/l2a_to_l2b.py

```diff
@@ -58,6 +58,39 @@ def dark_subtraction(input_dataset, dark_frame):
 
     return darksub_dataset
 
+def flat_division(input_dataset, flat_field):
+    """
+    
+    Divide the dataset by the master flat field. 
+
+    Args:
+        input_dataset (corgidrp.data.Dataset): a dataset of Images (L2a-level)
+        flat_field (corgidrp.data.FlatField): a master flat field to divide by
+
+    Returns:
+        corgidrp.data.Dataset: a version of the input dataset with the flat field divided out
+    """
+    
+     # copy of the dataset
+    flatdiv_dataset = input_dataset.copy()
+    
+    #Divide by the master flat
+    flatdiv_cube = flatdiv_dataset.all_data /  flat_field.data
```

Comment:
How are zero values in the master flat handled?  Are they special-cased in the division step, or precleaned/validated to ensure they can't exist, or...

Reply:
Currently, I don't think I am considering having zero values in the master flat when I generate them. In the test function, I have—assert np.mean(flat_frame.data) == pytest.approx(1, abs=1e-2), which would be a check.
Do you have any other suggestions I could add to the flat_division function?